<def f='tensorflow/tensorflow/python/lib/core/py_exception_registry.h' l='65' type='void tensorflow::PyExceptionRegistry::PyExceptionRegistry()'/>
<use f='tensorflow/tensorflow/python/lib/core/py_exception_registry.cc' l='26' u='c' c='_ZN10tensorflow19PyExceptionRegistry4InitEP7_object'/>
